testsuite/tests/mgc/reconciliation/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Module containing tests for Reconciliation of MGC policies"""
+
+from testsuite.policy.dns_policy import DNSPolicy
+
+
+def dns_policy(openshift, name, parent, issuer, labels: dict[str, str] = None):  # pylint: disable=unused-argument
```

Comment:
Should satisfy pylint:
```suggestion
def dns_policy(openshift, name, parent, _issuer, labels: dict[str, str] = None):
```

Reply:
I like ignore more this case, where we literally do not use it, and it is on purposes